### ISSUE TITLE
Warning on build - DEPRECATION WARNING

### ISSUE THREAD

**Issue by jjgriff93** [akveo/nebular#1108]
### Issue type

**I'm submitting a ...**  (check one with "x")

* [X] bug report
* [ ] feature request

### Issue description

**Current behavior:**
When doing an `ng build`, get these warnings related to Bootstrap within the ngx-admin template:

```
WARNING: Invalid value for $grid-breakpoints: This map must be in ascending order, but key 'is' has value 400px which isn't greater than 1200px, the value of the previous key 'xl' !
         on line 16 of node_modules/bootstrap/scss/_functions.scss, in mixin `-assert-ascending`
         from line 201 of node_modules/bootstrap/scss/_variables.scss
         from line 8 of node_modules/@nebular/bootstrap/styles/_globals.scss
         from line 7 of stdin

WARNING: Invalid value for $container-max-widths: This map must be in ascending order, but key 'is' has value 380px which isn't greater
than 1140px, the value of the previous key 'xl' !
         on line 16 of node_modules/bootstrap/scss/_functions.scss, in mixin `-assert-ascending`
         from line 221 of node_modules/bootstrap/scss/_variables.scss
         from line 8 of node_modules/@nebular/bootstrap/styles/_globals.scss
         from line 7 of stdin
```

**Expected behavior:**
No build warnings

**Comment by juliusiqbal** [comment]
Same here, with a bit change in output,

`
DEPRECATION WARNING on line 1, column 8 of D:/Important/projects/itlogiko/pnp-web/node_modules/@nebular/theme/components/cdk/overlay/_overlay.theme.scss:
Including .css files with @import is non-standard behaviour which will be removed in future versions of LibSass.
Use a custom importer to maintain this behaviour. Check your implementations documentation on how to create a custom importer.

WARNING: Invalid value for $grid-breakpoints: This map must be in ascending order, but key 'is' has value 400px which isn't greater than 1200px, the value of the previous key 'xl' !
         on line 16 of node_modules/bootstrap/scss/_functions.scss, in mixin `-assert-ascending`
         from line 201 of node_modules/bootstrap/scss/_variables.scss
         from line 8 of node_modules/@nebular/bootstrap/styles/_globals.scss
         from line 8 of node_modules/@nebular/bootstrap/styles/prebuilt/corporate.scss
         from line 12 of stdin

WARNING: Invalid value for $container-max-widths: This map must be in ascending order, but key 'is' has value 380px which isn't greater than 1140px, the value of the previous key 'xl' !
         on line 16 of node_modules/bootstrap/scss/_functions.scss, in mixin `-assert-ascending`
         from line 221 of node_modules/bootstrap/scss/_variables.scss
         from line 8 of node_modules/@nebular/bootstrap/styles/_globals.scss
         from line 8 of node_modules/@nebular/bootstrap/styles/prebuilt/corporate.scss
         from line 12 of stdin
`

**Comment by boehmmatthias** [comment]
Same problem here on ng serve or ng build

`WARNING: Invalid value for $grid-breakpoints: This map must be in ascending order, but key 'is' has value 400px which isn't greater than 1200px, the value of the previous key 'xl' !
         on line 16 of node_modules/bootstrap/scss/_functions.scss, in mixin `-assert-ascending`
         from line 201 of node_modules/bootstrap/scss/_variables.scss
         from line 8 of node_modules/@nebular/bootstrap/styles/_globals.scss
         from line 7 of stdin

WARNING: Invalid value for $container-max-widths: This map must be in ascending order, but key 'is' has value 380px which isn't greater than 1140px, the value of the previous key 'xl' !
         on line 16 of node_modules/bootstrap/scss/_functions.scss, in mixin `-assert-ascending`
         from line 221 of node_modules/bootstrap/scss/_variables.scss
         from line 8 of node_modules/@nebular/bootstrap/styles/_globals.scss
         from line 7 of stdin`

**Comment by Iloivar** [comment]
Same Problem:

WARNING: Invalid value for $grid-breakpoints: This map must be in ascending order, but key 'is' has value 400px which isn't greater than 1200px, the value of the previous key '
xl' !
         on line 16 of node_modules/bootstrap/scss/_functions.scss, in mixin `-assert-ascending`
         from line 201 of node_modules/bootstrap/scss/_variables.scss
         from line 9 of node_modules/@nebular/theme/styles/global/bootstrap/_bootstrap.theme.scss
         from line 7 of node_modules/@nebular/theme/styles/global/_overrides.scss
         from line 8 of node_modules/@nebular/theme/styles/_globals.scss
         from line 6 of stdin

WARNING: Invalid value for $container-max-widths: This map must be in ascending order, but key 'is' has value 380px which isn't greater than 1140px, the value of the previous key 'xl' !
         on line 16 of node_modules/bootstrap/scss/_functions.scss, in mixin `-assert-ascending`
         from line 221 of node_modules/bootstrap/scss/_variables.scss
         from line 9 of node_modules/@nebular/theme/styles/global/bootstrap/_bootstrap.theme.scss
         from line 7 of node_modules/@nebular/theme/styles/global/_overrides.scss
         from line 8 of node_modules/@nebular/theme/styles/_globals.scss
         from line 6 of stdin

**Comment by yggg** [comment]
I can't see `WARNING: Invalid value for $grid-breakpoints:` warning during the build. Tried on sources from the master or with the latest published package (3.3.0). Is somebody still facing this warning?

Regarding `DEPRECATION WARNING Including .css files with @import is non-standard behavior which will be removed in future versions of LibSass.`. `node-sass` developers changed their mind and allowed import of CSS files in latest versions. So we need to wait until the release of angular packages which use updated node-sass. Looks like it would be Angular 8.

**Comment by yggg** [comment]
Fixed in Angular 8
